### ISSUE TITLE
vscode metals not updating as download fails with ssl errors

### ISSUE THREAD

**Issue by fwaris** [scalameta/metals-vscode#1430]
**Describe the bug**
metals not updating due ssl certificate issues

**Screenshots**

Java home: C:\Program Files\Microsoft\jdk-17.0.8.101-hotspot\
Metals version: 1.0.1
Downloading https://repo1.maven.org/maven2/org/scalameta/metals_2.13/1.0.1/metals_2.13-1.0.1.pom
[...]
Downloading https://oss.sonatype.org/content/repositories/snapshots/org/scalameta/metals_2.13/1.0.1/metals_2.13-1.0.1.pom.sha1
Resolution error: Error downloading org.scalameta:metals_2.13:1.0.1
  not found: C:\Users\fwaris1\.ivy2\local\org.scalameta\metals_2.13\1.0.1\ivys\ivy.xml
  download error: Caught javax.net.ssl.SSLHandshakeException (PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://repo1.maven.org/maven2/org/scalameta/metals_2.13/1.0.1/metals_2.13-1.0.1.pom
  download error: Caught javax.net.ssl.SSLHandshakeException (PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target) while downloading https://dl.bintray.com/scalacenter/releases/org/scalameta/metals_2.13/1.0.1/metals_2.13-1.0.1.pom

**Installation:**

- Operating system: Windows
- VSCode version: latest as of 10/5
- VSCode extension version: 
- Metals version: (found in VSCode settings, under `metals.serverVersion`)

**Additional context**

I have update java version to latest

**Comment by tgodzik** [comment]
Thanks for reporting! To sure what might exactly be wrong, but you could try adding `-noverify` to `metals.serverProperties` as a workaround. A similar issue happened here https://github.com/coursier/coursier/issues/1977

**Comment by fwaris** [comment]
I think the issue is with our networking. The update does not work under a vpn connection but works outside of it

**Comment by wpopielarski** [comment]
@tgodzik unfortunately the trick with `-noverify` is not working for metals extension in vscode behind vpn.

**Comment by tgodzik** [comment]
We've had similar issue in Scala CLI more recently. Maybe something in https://github.com/VirtusLab/scala-cli/issues/2191 might help?

**Comment by fwaris** [comment]
in my case, the issue was related to the zscaler proxy (which intercepts ssl). To fix I had add the ZScaler root SSL cert to the java keystore. Another option is to use the option -Djavax.net.ssl.trustStoreType=WINDOWS-ROOT to make jdk use the windows store
________________________________

**Comment by wpopielarski** [comment]
so to install metals 1.3.1 and bloop 1.5.17 in viscose I did steps:
1. decrease metals extension version to 1.27 to allow crousier to load newest artifacts (to skip that certificate annoying problem, I had set -noverify but not sure if any impact of it)
2. install metals 1.3.1
3. restore version of extension back to 1.34.0
4. so in this point I faced problem with starting of bloop server so in step
5. restart Mac
6. coffee after 4 hours of investigation
